Makefile

```diff
@@ -55,11 +55,14 @@ BENCH_OUTPUT ?= stdout
 ALL_EXTENSIONS = debug,imagetrust,lint,metrics,mgmt,profile,scrub,search,sync,ui,userprefs
 EXTENSIONS ?= sync,search,scrub,metrics,lint,ui,mgmt,profile,userprefs,imagetrust
 UI_DEPENDENCIES := search,mgmt,userprefs
+# windows is not supported for pie builds at all
```

Comment:
interesting, has zot been tested for Windows at all?

Reply:
I just tried to build it on windows to check if there are no compile errors. However zot cant build on windows anyways.

I can remove this, if you want.

Reply:
Yes pls. Don't want folks getting ideas :)

Also, still early in the fixing-the-CI process.